udata/harvest/backends/base.py

```diff
@@ -84,11 +84,8 @@ def perform_initialization(self):
             if not self.dryrun:
                 self.job.save()
         except Exception as e:
-            log.error("Error in initialization : %s" % (str(e)))
-            log.exception(e)
             self.job.status = 'failed'
-            error = HarvestError(message=str(e),
-                                 details=traceback.format_exc())
```

Comment:
Ok. I have to say I was happy to find the traceback in the admin interface but if we have it in sentry I guess it's ok.